src/control/quickaction.cpp

```diff
@@ -45,7 +45,7 @@ void QuickAction::trigger() {
     }
 }
 
-void QuickAction::slotTriggered(double d) {
+void QuickAction::slotTriggered(double) {
```

Comment:
Usually we use `Q_UNUSED(d)` here.

Reply:
Is there a technical reason? Or just convention?

Reply:
Just convention.